documentation/docs/30-advanced/60-images.md

```diff
@@ -0,0 +1,149 @@
+
+You can mix and match both solutions in one project. For example, you may display images on your homepage with `@sveltejs/enhanced-img` and display user-submitted content with a dynamic approach.
+
+## @sveltejs/enhanced-img
+
+> **WARNING**: The `@sveltejs/enhanced-img` package is experimental. It uses pre-1.0 versioning and may introduce breaking changes with every minor version release.
+
+`@sveltejs/enhanced-img` offers plug and play image processing that serves smaller file formats like `avif` or `webp`, automatically sets the intrinsic width and height of the image to avoid layout shift, creates images of multiple sizes for various devices, and strips EXIF data for privacy. It will work in any Vite-based project including, but not limited to, SvelteKit projects.
+
+### Setup
+
+Install:
+
+```bash
+npm install --save-dev @sveltejs/enhanced-img
+```
+
+Adjust `vite.config.js`:
+
+```diff
+import { sveltekit } from '@sveltejs/kit/vite';
++import { enhancedImages } from '@sveltejs/enhanced-img';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	plugins: [
++		enhancedImages(),
+		sveltekit()
+	]
+});
+```
+
+### Basic usage
+
+Use in your `.svelte` components by using `enhanced:img` rather than `img` and referencing the image file with a [Vite asset import](https://vitejs.dev/guide/assets.html#static-asset-handling) path:
+
+```svelte
+<enhanced:img src="./path/to/your/image.jpg" alt="An alt text" />
+```
+
+At build time, your `img` tag will be replaced with a `picture` providing multiple image types and sizes. It's only possible to downscale images without losing quality, which means that you should provide the highest resolution image that you need — smaller versions will be generated for the various device types that may request an image. If you're not using the [`sizes` attribute](#sveltejs-enhanced-img-srcset-and-sizes) you should provide your image at 2x resolution for HiDPI displays (a.k.a. retina displays).
+
+Since the `enhanced:img` tag will be replaced with a `picture` tag, you should refer to it as such within your `style` tag. Generally we recommend styling the `img` child of the `picture` tag rather than the `picture` tag itself to make it easier to switch between various solutions such as a dynamic CDN-based solution should you ever need to do so.
```

Comment:
If we add SVG support in future, there'll be no `<picture>` at all, so I think this could end up being a red herring. We should probably nudge people towards using a classname rather than targeting the element, it's cleaner and simpler all round

```suggestion
Since the `<enhanced:img>` element is converted to an `<img>`, you can style it with an `img {...}` CSS rule, but you may find it more natural to add a `class` name and target that.
```